discord/abc.py

```diff
@@ -441,8 +441,21 @@ def permissions_synced(self):
 
         .. versionadded:: 1.3
         """
+        def remove_empty_permissions(overwrite):
+            new_overwrite = [] # Able to keep it as a list since this is never getting passed
+                               # to the user.
+            for ow in overwrite: 
+                if not ((ow.allow == 0) and (ow.deny == 0)): # Only compare permissions that
```

Comment:
```suggestion
                if not (ow.allow == 0 and ow.deny == 0): # Only compare permissions that
```
These parentheses are redundant.

Reply:
Technically all of the parentheses are redundant but removing all of them might make the code hard to read.